repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/IdentityStatus.java

```diff
@@ -22,19 +22,12 @@
 /**
  * @author fran
  */
-public enum IdentityStatus {
+public class IdentityStatus {
```

Comment:
I agree. This one could have gone either way. It just seems kind of heavyweight as an `enum` when it is just being used as `String`'s.